pkg/gds/admin/v2/api.go

```diff
@@ -39,6 +40,17 @@ type StatusReply struct {
 // Admin v2 API Requests and Responses
 //===========================================================================
 
+// Summary provides aggregate statistics that describe the state of the GDS.
+type SummaryReply struct {
+	VASPsCount           int            `json:"vasps_count"`           // the total number of VASPs in any state in GDS
+	PendingRegistrations int            `json:"pending_registrations"` // the number of registrations pending (in any pre-review status)
+	ContactsCount        int            `json:"contacts_count"`        // the number of contacts in the system
+	VerifiedContacts     int            `json:"verified_contacts"`     // the number of verified contacts in the system
+	CertificatesIssued   int            `json:"certificates_issued"`   // the number of certificates issued by the GDS
+	Statuses             map[string]int `json:"statuses"`              // the counts of all statuses in the system
```

Comment:
"Registration" and "Verification" can be used as synonyms; the nuance is that a registration is the client-side perspective, e.g. they've submitted a registration; whereas verification is action that a reviewer is taking. I've been trying to minimize my use of "verification" since it gets confusing (hence the short hand for statuses); for example a "verified contact" is one that has clicked a link with a token that was sent to their email.